src/main/java/frc/robot/subsystems/NeoMotorDriveSystem.java

```diff
@@ -0,0 +1,112 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class NeoMotorDriveSystem extends SubsystemBase 
+{
+
+  ///CONSTANTS///
+  private static boolean debugMode = true;
+
+  // IDs
+  private static final int frontLeft_ID  = 2;
+  private static final int frontRight_ID = 4;
+  private static final int backLeft_ID   = 3;
+  private static final int backRight_ID  = 5;
+
+  // Invert
+  private static final boolean invertLeft  = true;
+  private static final boolean invertRight = false;
+
+  // Motors
+  private CANSparkMax m_frontLeftMotor;
+  private CANSparkMax m_frontRightMotor;
+  private CANSparkMax m_backLeftMotor;
+  private CANSparkMax m_backRightMotor;  
+
+  private final RelativeEncoder m_frontLeftEncoder;
+  private final RelativeEncoder m_frontRightEncoder;
+  private final RelativeEncoder m_backLeftEncoder;
+  private final RelativeEncoder m_backRightEncoder;
```

Comment:
I want to test if the encoders are actually outputting to the SmartDashboard (if so, then we can keep these lines)